### ISSUE TITLE
Run model 2 with tensorflow for watershed data (from 2017/02/23 ~ )

### ISSUE THREAD

**Issue by tomowind** [garlicbulb-puzhuo/ggo#56]

**Comment by tomowind** [comment]
Running from 02/23 12AM. 
Running from sangmin-tf-ontopof-20170222 branch.

Here's the config...

```
[config]
img_rows=512
img_cols=512
model_id=2
batch_size=2
nb_epoch=500
standalone_loss_history_file=standalone_loss_history_file
```

**Comment by tomowind** [comment]
Result (02/26): 
- 45 epochs. 
- Loss not going down. 
- Ran more than 3 days.
- Saved weights and losses.

Next: 
- Load theano's weight? (try this: http://stackoverflow.com/questions/42193436/keras-convert-pretrained-weights-between-theano-and-tensorflow)
- Use tensorflow's weight to theano

**Comment by tomowind** [comment]
@oakcreek some help?

**Comment by tomowind** [comment]
Add results to code in results directory in the following form:

`results/model_2_tensorflow.md`

```
- configuration: (write this if you modified the source to run the model)
- code: which branch, which commit
- result: 
  - loss: something
  - ...
- weight file: saved in sangmin's machine
```
